CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # @digitalbazaar/ecdsa-xi-2023-cryptosuite Changelog
 
+## 1.1.0 - 2024-02-15
+
+### Added
```

Comment:
I think this is a fix rather than a new feature -- and remember to just put `dd` for the date as it allows the package release manager to set it as part of the release process.

```suggestion
## 1.0.1 - 2024-02-dd

### Fixed
```